test/sampletests/globalReporterSkippedCount/skippedCount.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Please keep the existing naming convention, all lower case folder name and use a different name for the test file.